attractions/utils/events.js

```diff
@@ -1,4 +1,9 @@
-/* An action to set up arbitrary event listeners dynamically. */
+/**
+ * An action to set up arbitrary event listeners dynamically.
+ * @param {Element} node
+ * @param {Array<{name: string; handler: EventListenerOrEventListenerObject}>} args The event listeners to be registered
```

Comment:
```suggestion
 * @param {Array<{ name: string; handler: EventListenerOrEventListenerObject }>} args The event listeners to be registered
```

Reply:
Also not sure about that `EventListenerOrEventListenerObject`, what is that supposed to be? This thing should simply take callbacks, is that it?

Reply:
It is defined internally as `type EventListenerOrEventListenerObject = EventListener | EventListenerObject;`
`EventListener` is the regular callback you are used to, and `EventListenerObject` is an object that has the handler called `handleEvent`. Read more in the [MDN docs for the event listener callback](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#the_event_listener_callback).
They type of `node.addEventListener` is the following
```ts
Element.addEventListener(type: string, listener: EventListenerOrEventListenerObject, options?: boolean | AddEventListenerOptions): void
```
so I just used the same type for the listener we register.